rpc/grpc/config.go

```diff
@@ -26,9 +26,12 @@ import (
 // Config is a configuration for GRPC netListener
 // It is meant to be extended with security (TLS...)
 type Config struct {
-	// Endpoint is an address of GRPC netListener
+	// Endpoint is an address of GRPC netListener for TCP/IP transport. Always chosen as default if set.
 	Endpoint string `json:"endpoint"`
 
+	// UnixSocketFilePath defines a path for unix socket file.
+	UnixSocketFilePath string `json:"unix-socket-file-path"`
```

Comment:
Recommend we mirror how net.Listen works and simply specify the socketType (see https://github.com/ligato/cn-infra/pull/273 ).  What is the correct behavior if someone specifies both Endpoint and UnixSocketFilePath?